service/account/providers/aws/awscredential.go

```diff
@@ -0,0 +1,157 @@
+package aws
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/spotinst/spotinst-sdk-go/spotinst"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/client"
+	"github.com/spotinst/spotinst-sdk-go/spotinst/util/jsonutil"
+	"io/ioutil"
+	"net/http"
+)
+
+type Credential struct {
+	IamRole   *string `json:"iamRole,omitempty"`
+	AccountId *string `json:"accountId,omitempty"`
+
+	// Read-only fields.
+	//CreatedAt *time.Time `json:"createdAt,omitempty"`
+	//UpdatedAt *time.Time `json:"updatedAt,omitempty"`
```

Comment:
remove commented and unwanted line of codes

Reply:
removed.